suites/quincy/cephfs/tier-3_cephfs_scale.yaml

```diff
@@ -164,3 +173,11 @@ tests:
       module: cephfs_scale.max_clones_from_snap.py
       name: "Clone scale testing"
       polarion-id: "CEPH-83575629"
+  - test:
+      name: snap_retention_count_validate_subvol
```

Comment:
Name here needs to be modified from "snap_retention_count_validate_subvol" to "snap_retention_count_validate_vol"